app/__init__.py

```diff
@@ -1,3 +1,100 @@
-from flask import Flask
+from flask import Flask, jsonify, request
 
 app = Flask(__name__)
+
+
+requests = {}
+
+users = {}
+
+
+# POST a user
+@app.route('/users', methods=['POST'])
+def create_user():
+    user = {
+        "username": request.json.get("username"),
+        "email": request.json.get('email'),
+        "password": request.json.get("password"),
+        "cconfirm_password": request.json.get('cconfirm_password'),
+    }
+    users.update(user)
+    return jsonify({"users": users}), 201
+
+
+# GET a user
+@app.route('/users/<string:userId>')
+def get_user(userId):
+    user = [user for user in users if user["id"] == userId]
+    if len(user) == 0:
+        return jsonify({"message": "Not Found"}), 404
+
+    else:
+        return jsonify({"user": user}), 200
+
+
+# POST a request
+@app.route('/users/requests', methods=['POST'])
+def create_request():
+    # "id" = request.json.get("id")
+    request_data = {request.json.get("id"): {
```

Comment:
The json is structured wrong, it should be `{"id":len(requests),
        "title": request.json.get("title"),
        "type": request.json.get('type'),
        "description": request.json.get("description"),
        "category": request.json.get('category'),
        "area": request.json.get('area')
    }`